src/assets/scss/app.scss

```diff
@@ -5,3 +5,4 @@
 @import "./settings";
 // Load Foundation
 @import "~foundation-sites/scss/foundation";
+@include foundation-everything;
```

Comment:
This is the detail I missed then.

Reply:
Before I went on vacation, Liz mentioned being concerned about bloating the CSS, so this is also something we might want to pare down once we know what is being used.

Reply:
I think so, there were so many parts I just added them all for the time being.